MVend/guiza-dashboard-api
# bg():fix models naming

#### Problem
---
The migrated tables were started Uppercase in the source code and MySQL DB stores the tables with only lowercase letters.
This caused the issue to not find tables. Here is an example of an error that occurred
`SequelizeDatabaseError: Table 'test_core_gwiza_stagging.Admins' doesn't exist`
In the source code, I was using Admins and in DB was saved as admins

#### Description of changes made
---

- Renamed the models
- Apply new names to the controllers that consume these models
- Removed the migrations

#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/MVend/guiza-dashboard-api.git), and `cd [the project directory]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `npm i` to install the project dependencies
- Run `npm run dev` to start the server in the development mode.


#### Checklist:



#### Blocker
> **If there is any blocker you have just mention it below**
N/A

#### Screenshots
N/A
